src/main/java/com/firebolt/jdbc/connection/FireboltConnectionServiceSecret.java

```diff
@@ -102,19 +99,17 @@ protected void assertDatabaseExisting(String database) throws SQLException {
 
     private FireboltProperties getSessionPropertiesForSystemEngine(String accessToken, String accountName) throws SQLException {
         String systemEngineEndpoint = fireboltGatewayUrlService.getUrl(accessToken, accountName);
-        FireboltAccount account = fireboltAccountIdService.getValue(accessToken, accountName);
-        infraVersion = account.getInfraVersion();
+        infraVersion = 2;
         URL systemEngienUrl = UrlUtil.createUrl(systemEngineEndpoint);
         Map<String, String> systemEngineUrlUrlParams = UrlUtil.getQueryParameters(systemEngienUrl);
-        String accountId = systemEngineUrlUrlParams.getOrDefault(ACCOUNT_ID.getKey(), account.getId());
         for (Entry<String, String> e : systemEngineUrlUrlParams.entrySet()) {
             loginProperties.addProperty(e);
         }
         return loginProperties
                 .toBuilder()
                 .systemEngine(true)
                 .compress(false)
-                .accountId(accountId)
+                .accountId(null)
```

Comment:
Can we not remove accountid line here altogether since this is a builder? Or are we wiping it on purpose?

Reply:
Just left it like this for visibility. Also, I'm not sure what default value is set, so setting this explicitly made sense to me

Reply:
The default here is [null](https://github.com/firebolt-db/jdbc/blob/9daa67fc6274e339e6941438202af12412374044/src/main/java/com/firebolt/jdbc/connection/settings/FireboltSessionProperty.java#L63) so let's remove this line to avoid unnecessary complexity.